### DISCUSSION TITLE
ReLU op

### DISCUSSION THREAD

**Discussion by finncatling** [pymc-devs/pytensor#401] [Q&A] [answered]
Hi, thanks for your work on this library.

I would like to use a ReLU op within my pymc model. I can't find one within pytensor since the removal of `pytensor.tensor.nnet` so am currently relying on `aesara.tensor.nnet.basic.relu`. However, `aesara.tensor.nnet` is deprecated and I believe will soon be removed.

Are there any plans to add this functionality back to pytensor, please? Thanks again.

**Comment by ricardoV94** [comment, marked as answer]
You can create a `relu` with `pytensor.tensor.switch`:

```python
import pytensor.tensor as pt

def relu(x):
  return pt.switch(pt.lt(x, 0), 0, x)
```

No need for a custom Op

**Reply by finncatling** [reply]
Many thanks!
